prow/repoowners/repoowners.go

```diff
@@ -145,8 +145,8 @@ type RepoOwnerInterface interface {
 	RequiredReviewers(path string) sets.String
 }
 
-// RepoOwners implements RepoOwnerInterface
-var _ RepoOwnerInterface = &RepoOwners{}
+// RepoOwners implements RepoOwner
```

Comment:
This is a golang trick to ensure that RepoOwners implements the RepoOwner interface correctly, otherwise we'll get a compile error

Reply:
Not sure why I added a comment here. 😅